### DISCUSSION TITLE
Developer meeting (May 8th): Prototype of forward facing expansion connector

### DISCUSSION THREAD

**Discussion by evoggy** [bitcraze/discussions#1315] [Announcements]
The next developer meeting will be on the 8th of May 2024, we will talk about a new prototype of forward facing expansion connector.

Date: Wednesday, May 8th, 2024
Time: 3 PM Central European Summer Time (check your [timezone](https://www.worldtimebuddy.com/?qm=1&lid=2692969,5368361,1850147&h=2692969&date=2024-05-08&sln=15-16&hf=0))
Topic: Prototype of forward facing expansion connector
Hosts: @evoggy 
Meeting link: Meeting is over! See the recording here https://youtu.be/4abvN_ZUV2U
Slides from meeting: [link to PDF](https://github.com/bitcraze/discussions/files/15330601/Dev.meeting.240508.-.Owl.pdf)

15:00 - 15:30: Presentation and discussion
15:30 - 16:00: Generic support questions

Indicate with 👍 if you are planning to join!

![IMG_20240429_111850](https://github.com/bitcraze/discussions/assets/5947128/5b8a93ce-6db6-460a-b39a-d3cae6544487)

**Comment by knmcguire** [comment]
Recording can be found here : https://youtu.be/4abvN_ZUV2U

**Reply by Kazimbalti** [reply]
Hello @knmcguire @evoggy are you in the icra2024 conference, and you will demonstrate this forward facing expansion camera there? thank you

**Reply by knmcguire** [reply]
Hi! Me and Marcus aren't there but Arnaud, barbara and Rik are there, with the forward facing connector prototype on display. They are located at booth IC085

**Comment by Tesseract-Developer** [comment]
@knmcguire Would you post the corresponding presentation from the video here as well?

**Reply by knmcguire** [reply]
didn't think people would be interested, but it seems I'm wrong! I'll ask Marcus to upload them this week.

**Reply by evoggy** [reply]
I've added a link to the slides in the original post above.
